common/beerocks/tlvf/AutoGenerated/src/beerocks/tlvf/beerocks_message_backhaul.cpp

```diff
@@ -41,8 +41,8 @@ char* cACTION_BACKHAUL_REGISTER_REQUEST::sta_iface(size_t length) {
 
 bool cACTION_BACKHAUL_REGISTER_REQUEST::set_sta_iface(const std::string& str) { return set_sta_iface(str.c_str(), str.size()); }
 bool cACTION_BACKHAUL_REGISTER_REQUEST::set_sta_iface(const char str[], size_t size) {
-    if (str == nullptr || size == 0) {
-        TLVF_LOG(WARNING) << "set_sta_iface received an empty string.";
+    if (str == nullptr) {
```

Comment:
Since you anyway change this line (and all the others), you could have gone for `if(!str)` but ok...

Reply:
This was intentional. It's not compact, but a bit easier to read, also the search for "str == nullptr" showed that it is more common for this codebase.

Reply:
See https://github.com/prplfoundation/prplMesh/blob/master/CODINGSTYLE.md#klocwork - KW complains about this, so we decided to opt for `if (!ptr)` over `if (`ptr == nullptr)`.
We don't run KW on the autogenerated code yet so its not an issue, however if we can fix some things along the way then we certainly should.